platform/api/aws/images.go

```diff
@@ -289,3 +365,147 @@ func registerImageParams(snapshotID, name, description string, diskBaseName stri
 		},
 	}
 }
+
+type copyImageResult struct {
+	region  string
+	imageID string
+	err     error
+}
+
+func (a *API) CopyImage(sourceImageID string, regions []string) (map[string]string, error) {
+	image, err := a.describeImage(sourceImageID)
+	if err != nil {
+		return nil, err
+	}
+
+	describeSnapshotRes, err := a.ec2.DescribeSnapshots(&ec2.DescribeSnapshotsInput{
+		SnapshotIds: []*string{image.BlockDeviceMappings[0].Ebs.SnapshotId},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("couldn't describe snapshot: %v", err)
+	}
+	snapshot := describeSnapshotRes.Snapshots[0]
+
+	var count int
```

Comment:
I think a waitgroup expresses the intent of count more clearly than an int.

The channel could then turn into a buffered channel that's closed on `wg.Done` and ranged over.